DRAFT: Multi-Party Payout.md

```diff
@@ -0,0 +1,204 @@
+<pre>
+  DIP: 000x
+  Title: Multi-Party Payouts
+  Author(s): Timothy Munsell,
+  Comments-Summary: No comments yet.
+  Status: Draft
+  Type: Standard
+  Created: 2021-05-08
+  License: MIT License
+</pre>
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Abstract](#abstract)
+- [Motivation and Previous System](#motivation-and-previous-system)
+  - [Problems with the previous system](#problems-with-the-previous-system)
+- [Prior Work](#prior-work)
+- [New Stake Holder Reward Fields within ProRegTx and ProUpRegTx](#new-stake-holder-reward-fields-within-proregtx-and-proupregtx)
+- [Registering a Masternode (ProRegTx)](#registering-a-masternode-proregtx)
+- [Updating Masternode Information](#updating-masternode-information)
+  - [Updating Registrar of Masternode (ProUpRegTx)](#updating-registrar-of-masternode-proupregtx)
+- [Validation Rules](#validation-rules)
+  - [ProRegTx and ProUpRegTx](#proregtx-and-proupregtx)
+  - [ProUpRevTx](#prouprevtx)
+- [Masternode Rewards](#masternode-rewards)
+- [Copyright](#copyright)
+
+## Abstract
+
+This DIP builds on the chain consensus for masternode lists laid forth in DIP0003 and provides for multi-party payout beyond the single owner + operator framework of the current system.
+
+## Motivation and Previous System
+
+In the previous system, Masternodes gained entry to the masternode list after the owner created a ProRegTx (DIP3). This tx provided key IDs for up-to 2 roles that would receive masternode rewards payouts:
+
+ * Owner
+ * Operator
+
+Masternode rewards are payed to the Owner by default, and when defined, to the Operator. Current updatesto the masternode registrar  (relevant to this DIP) could only:
+
+  * Define new Operator
+  * Define payout to Operator
+
+### Problems with the previous system
+
+While elegant, under the current system, the masternode reward is paid to the operator based on the `<operatorReward>` field and any difference from 100% is automatically paid to the owner address.
+
+The limitation of rewards paid to only 2 addresses based on a single value field prevents automatic/trustless sharing of the rewards beyond the owner and/or operator, a critical element for enabling trustless shares in a masternode and introducing Dash native DeFi opportunities through staking (among other use cases).
+
+## Prior Work
+
+* [DIP 0002: Special Transactions](https://github.com/dashpay/dips/blob/master/dip-0002.md)
+* [DIP 0003: Deterministic Masternode Lists](https://github.com/dashpay/dips/blob/master/dip-0003.md)
+
+## New Stake Holder Reward Fields within ProRegTx and ProUpRegTx
+
+In the new system the ProRegTx and ProUpRegTx (DIP3) may include an additional fields to define additional parties to receive rewards beyond the owner and operator and the % of owner rewards each party receives.

```

Comment:
- comma after "In the new system"
- ~an~ additional fields
- "other parties" may work better here